src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -235,10 +235,14 @@ public function getCrumbs()
      * - *innerAttrs* To provide attributes in case your separator is divided in two elements.
      * All other properties will be converted as HTML attributes and will replace the *attrs* key in the template.
      * If you use the default for this option (empty), it will not render a separator.
-     * @return string The breadcrumbs trail
+     * @return string|null The breadcrumbs trail
      */
     public function render(array $attributes = [], array $separator = [])
     {
+        if (empty($this->crumbs)) {
```

Comment:
See http://book.cakephp.org/3.0/en/contributing/cakephp-coding-conventions.html#careful-when-using-empty-isset
So `if (!$this->crumbs)` would always be better as it doesnt hide spelling mistakes etc.